packages/@okta/vuepress-site/docs/reference/api/features/index.md

```diff
@@ -0,0 +1,696 @@
+
+If you use the `mode=force` parameter while enabling a Feature, then Okta first tries enabling any disabled Features that this Feature may have as dependencies. If you do not pass the `mode=force` parameter and the Feature has dependencies that need to be enabled before the Feature is enabled, a `400` error is returned.
+
+If you use the `mode=force` parameter while disabling a Feature, then Okta first tries disabling any enabled Features that this Feature may have as dependents. If you do not pass the `mode=force` parameter and the Feature has dependencies that need to be disabled before the Feature is disabled, a `400` error is returned.
+
+#### Request Body
+
+None
+
+#### Response Body
+
+Updated [Feature](#feature-model)
+
+#### Example Usage
+
+The following request enables the Feature with an `id` value `ftrZooGoT8b41iWRiQs7`.
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+}' "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/enable"
+```
+
+##### Response
+
+```json
+{
+  "id": "ftrZooGoT8b41iWRiQs7",
+  "type": "self-service",
+  "status": "ENABLED",
+  "name": "Example feature name",
+  "description": "Example feature description",
+  "stage": {
+    "value": "EA"
+  },
+  "_links": {
+    "disable": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/disable",
+      "hints": {
+        "allow": [
+          "POST"
+        ]
+      }
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7"
+    },
+    "dependents": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/dependents"
+    },
+    "dependencies": {
+      "href": "https://{yourOktaDomain}/api/v1/features/ftrZooGoT8b41iWRiQs7/dependencies"
+    }
+  }
+}
+```
+
+##### Error Response (Invalid featureId)
+
+An invalid `featureId` returns a `404 Not Found` status code.
+
+```json
+HTTP/1.1 404 Not Found
+Content-Type: application/json
+
+{
+    "errorCode": "E0000007",
+    "errorSummary": "Not found: Resource not found: invalidFeatureId (Feature)",
+    "errorLink": "E0000007",
+    "errorId": "oaenAlD87mGTuWeGi7AjmhIBA",
+    "errorCauses": []
+}
+```
+##### Error Response (Method not allowed)
+
+The following returns a `405 Method Not Allowed` status code:
+
+* An invalid `lifecycle` value
+* Update request for a `BETA` Feature in a non-preview cell
+* If the Feature requires support to be updated
+
+```json
+HTTP/1.1 405 Method Not Allowed
+Content-Type: application/json
+
+{
+    "errorCode": "E0000022",
+    "errorSummary": "The endpoint does not support the provided HTTP method",
+    "errorLink": "E0000022",
+    "errorId": "oaerzuJCOHVQ8-E2XpkEKfeOQ",
+    "errorCauses": []
+ }
+```
+
+##### Error Response (Already disabled/enabled)
```

Comment:
I still have the same doubt/question about this one. If we're doing the same for other APIs, then fine. I don't say anything in docs for User Lifecycle APIs for example so I cannot be sure. If we're not doing the same, then we have to revise this decision. Semantically speaking this is more of a 304 situation to me but I guess 405 is also applicable especially if it means keeping consistent with the rest of APIs: https://developer.okta.com/docs/reference/api/users/#lifecycle-operations

Reply:
had a call about this with Tom. so he suggested that 405 is better and more consistent with other endpoints for this case than 304. We're not returning a 304 in any other endpoint cases.